fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/digitalcash/DigitalCashReceiveFragment.java

```diff
@@ -0,0 +1,48 @@
+package com.github.dedis.popstellar.ui.digitalcash;
+
+import android.os.Bundle;
+
+import androidx.fragment.app.Fragment;
+
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.github.dedis.popstellar.R;
+
+/**
+ * A simple {@link Fragment} subclass.
+ * Use the {@link DigitalCashReceiveFragment#newInstance} factory method to
+ * create an instance of this fragment.
+ */
+public class DigitalCashReceiveFragment extends Fragment {
+
+    public DigitalCashReceiveFragment() {
+        // Required empty public constructor
+    }
```

Comment:
is this necessary? Doesn't Java offer a by default empty constructor implicitly if we don't have a any constructor? I would tend to remove this, but you might have a different opinion?

Reply:
Maybe keep it for Hilt in the future?